### DISCUSSION TITLE
why i alway fail Connection error while connecting to

### DISCUSSION THREAD

**Discussion by art1469** [TECH7Fox/Asterisk-integration#67] [Q&A]
i don't know about detail that i wanna filed

how to see port , user ,pass i try many user but alway cannot connect 

Help Me pls

**Comment by TECH7Fox** [comment]
You need the AMI user. Do you use the add-on?
Otherwise, you can see the login info in manager.conf. And also make sure that your IP is allowed in there.

**Reply by art1469** [reply]
i must to install Astreisk Add-on First Right?  Add-on , intergration and Sip-Card tatal 3 method i must install Right?

**Reply by TECH7Fox** [reply]
Yes. The integration is optional. In the docs is some more info.
https://sip-hass-docs.github.io/sip-hass-docs/
